runtime/doc/builtin.txt

```diff
@@ -8714,7 +8716,7 @@ win_gettype([{nr}])					*win_gettype()*
 		popup window then 'buftype' is "terminal" and win_gettype()
 		returns "popup".
 
-		Return an empty string if the window cannot be found.
+		Returns "unknown" if the window cannot be found.
```

Comment:
Turns out it's already mentioned a bit above, so this can be removed.

Also the references above to popup windows look weird.
Nvim returns "popup" for floating windows instead. We should refer to them as that and link to |api-floatwin| instead.